### ISSUE TITLE
Pagination does not work with loop is set to true.

### ISSUE THREAD

**Issue by martinjinda** [nolimits4web/Swiper#2245]
This is a (multiple allowed):
* [x] bug

* Swiper Version: 4.0.0
* Platform/Target and Browser Versions: macOs
* Webpack: 3.6.0

### What you did
When you turn on loop setting, pagination will stop working.

```
let swiper = new Swiper('.swiper-container', {
	pagination: {
		el: '.swiper-pagination',
		clickable: true,
	},
	loop: true,
	speed: 1500,
	autoplay: {
		enabled: true,
		delay: 5000,
		disableOnInteraction: false,
	}
});
```

**Comment by iwata** [comment]
I reproduced this issue on v4.0.1, too.

**Comment by nolimits4web** [comment]
From what i see here it is working http://idangero.us/swiper/demos/200-infinite-loop.html

**Comment by zhansingsong** [comment]
I got the same issue on 4.0.7. I suggest to reopen the issue

**Comment by filippozanardo** [comment]
same in 4.1.0 with:
`swipergallery = new Swiper('#pjswiper', {
	    slidesPerView: 1,
	  	loop: false,
	    parallax: true,
	    grabCursor: false,
	    observer: true,
		observeParents: true,
	    speed: 1500,
	    pagination: {
	    	el: '.swiper-pagination',
	    	clickable: true,
	  	},
	    navigation: {
	    	nextEl: '.swiper-button-next',
	    	prevEl: '.swiper-button-prev',
	  	},
	  	
	  });`

**Comment by rimosdev** [comment]
have the same issue

**Comment by xllily** [comment]
I got the same issue on **4.3.0.** 
The swiper-container is `display:none;` when `new Swiper();`.

When container displayed, `".swiper-pagination.swiper-pagination-bullets"` is empty node.
And the console did not report any errors.

Then I switched to version **3.4.2**,

just changed like
`pagination: {
    el: '.swiper-pagination',
  },`  
to
 `pagination '.swiper-pagination',` 

**then working fine**.
